app/views/posts/index.html.erb

```diff
@@ -1,12 +1,22 @@
-<% if @user %>
-  <h1><%= @user.name %>'s Posts</h1>
-<% end %>
+<%= render 'shared/user_profile', user: @user %>
 
-<ul>
-  <% @posts.each do |post| %>
-    <li>
-      <%= post.title %>
-      <%= link_to 'Delete', delete_post_path(post), method: :delete, data: { confirm: 'Are you sure you want to delete this post?' } %>
-    </li>
-  <% end %>
-</ul>
+<% @posts.each  do  |post|%>
+<div class="boxed comments_container">
+    <a href="/users/<%= post.author_id%>/posts/<%= post.id %>"><%= post.title%></a>
+        <p>
+           <%= post.text%>
+        </p>
+        <p class="comments_likes">
+            Comments: <%= post.comment_counter %> Likes: <%= post.likes_counter%>
+        </p>
+</div>
+<div class="boxed">
+<% Post.get_5_comments(post).each  do |comment|%>
```

Comment:
- [ ] To ensure that only 5 recent comments are displayed:

![image](https://github.com/adarachel/rails-blog/assets/12745474/2bc246e4-9e48-4071-902a-1614df646c98)